stories/alien/level1/slides.html

```diff
@@ -401,57 +456,62 @@
 
 ### Sequencing
 
-![https://www.pinterest.ru/pin/107664247327936161/](images/dna-tetris.png)
+![DNA tetris](images/dna-tetris.png)
 
 <small><small>[https://www.pinterest.ru/pin/107664247327936161/]</small></small>
 
 ???
--In Sam's mind, he wondered: can it be determined in which order the nucleotides occur in every type of organism? 
--When you have a small piece of material from a specific organism, can you identify this order?
+-In Sam's mind, he is wondering: can it be determined in which order the nucleotides occur in every type of organism? 
+-When you have a small piece of material from a specific organism, can you identify the sequence of its bases?
+-An automated system would be clever, because reading the whole DNA of e.g. a human which has 6.4 billion letters would take Sam too much time.
+
 
 ---
 
 ### Sequencing
 
-![https://www.pinterest.ru/pin/10485011602449279/](images/sequencing1.png)
+![Sequencing](images/sequencing1.png)
 
 <small><small>[https://www.pinterest.ru/pin/10485011602449279/]</small></small>
 
 ???
--After some research, he realized that a technique for determining this order already exists. 
+-After some research, he realizes that a technique for determining this order already exists. 
 -This technique is called DNA sequencing.
+-And using sequencing one can determine if a DNA stretch originates from a house human or gorilla just by reading out the sequences of the bases.
 
 ---
 
 ### Sequencing
 
-![https://www.pinterest.ru/pin/88453580171419743/](images/dna-4-hands.png)
+![Four hands building DNA](images/dna-4-hands.png)
 
 <small><small>[https://www.pinterest.ru/pin/88453580171419743/]</small></small>
 ???
--DNA sequencing is the process of determining the nucleic acid sequence – the order of nucleotides in DNA. 
```

Comment:
I am not sure if we need this picture. maybe the information can be added to the two slides before. Don't know if the hands help to understand that there are 4 bases. Have we talked before about the bases? Then better use this part.

Reply:
Makes sense, this picture is ambiguous, I added the info to the slide before.